code/game/objects/items/cards_ids.dm

```diff
@@ -435,6 +435,17 @@ update_label("John Doe", "Clowny")
 				to_chat(usr, "<span class='notice'>The card's microscanners activate as you pass it over the ID, copying its access.</span>")
 
 /obj/item/card/id/syndicate/attack_self(mob/user)
+	var/static/valid_jobs = list( // copy-paste from hop card paint machine, but this has more options, and valid to be static.
```

Comment:
I really hate this, there should be a better system or UI here as a future PR, the ID painter should use TGUI or something instead of an ugly pick() text list. Maybe images of all the cards, split into tabs by department, labeled with the job name? Something fancy and nice.

Also job HUD names should be DEFINES as well, including the custom ones.

Reply:
> Also job HUD names should be DEFINES as well, including the custom ones.

ah, right. I forgot I changed it as job defines before.

> I really hate this, there should be a better system or UI here as a future PR, the ID painter should use TGUI or something instead of an ugly pick() text list. Maybe images of all the cards, split into tabs by department, labeled with the job name? Something fancy and nice.

that should be done in future PR along with HoP painting machine, not here. I don't think it's necessary for now.